src/components/battleItemContainer.html

```diff
@@ -1,4 +1,14 @@
 <div id="battleItemContainer" class="card sortable border-secondary mb-3" data-bind="visible: !(Settings.getSetting('hideChallengeRelatedModules').observableValue() && App.game.challenges.list.disableBattleItems.active())">
+    <button type="button" class="btn btn-sm btn-primary"
+            style="position: absolute; left: 0px; top: 0px; width: auto; height: 41px;"
+            onclick="ItemHandler.useAllBattleItems()"
+            title="Using all Battle Items"
+            data-toggle="popover"
+            data-html="true"
+            data-content="Click to use all items at the same time"
```

Comment:
For consistency, we should use the same tooltip, as we are using everywhere else (like the battle items)